tests/test.sh

```diff
@@ -32,7 +32,7 @@ expect() {
 
     echo "$expected_output" > "$tmp_expected"
 
-    "$fd" "$@" | sort -f > "$tmp_output"
+    "$fd" "$@" | xargs -0 echo -n | sort -f > "$tmp_output"
```

Comment:
That does work. The reason I didn't do that is that including `\0` causes git to treat `test.sh` as binary and then it's no longer diffable. Perhaps there's another way to resolve this, but using `xargs` seemed like the easiest solution.